packages/api/src/usePrefetchQuery.ts

```diff
@@ -0,0 +1,31 @@
+import { useQueryClient } from '@tanstack/react-query';
```

Comment:
are there any general standards of codebase documentation within deriv-app?

Reply:
No, we dont have any documentation yet for the general hooks

Reply:
We dont use the hooks anywhere yet. I added these hooks for future improvement

Reply:
YAGNI :) unless its actually used, typically better to leave it out,

Reply:
I created it because i wanted to use it in this PR, but the behaviour not as i expected. So, i just leave it there and not remove  it  :)

Reply:
is it expected to be used elsewhere? we shouldn't have dead code in codebase really,

Reply:
I can remove it for now, in case we need later, i will create it back